src/lib/settings/settings.py

```diff
@@ -116,19 +116,24 @@ def get_raw_settings(self, file_name: str) -> dict:
     # CDK methods
     def get_monitored_account_ids(self) -> List[str]:
```

Comment:
May only want to change the return type

Reply:
What return type would you consider as preferable?

Reply:
set[str]?